src/coreclr/vm/readytoruninfo.h

```diff
@@ -66,7 +66,7 @@ class ReadyToRunInfo
     PTR_CORCOMPILE_IMPORT_SECTION   m_pImportSections;
     DWORD                           m_nImportSections;
 
-    bool                            m_readyToRunCodeDisabled;
+    bool                            m_readyToRunCodeDisabled; // Is 
```

Comment:
Nit - looks like an unfinished comment. If the purpose of the one-word comment is just to indicate that the flag states whether ready to run code is disabled, would it be perhaps better to directly rename the identifier to something like <code>m_isReadyToRunCodeDisabled</code>?